tools/clang/lib/Parse/ParseDecl.cpp

```diff
@@ -1900,6 +1899,14 @@ Parser::DeclGroupPtrTy Parser::ParseDeclaration(unsigned Context,
     ProhibitAttributes(attrs);
     SingleDecl = ParseDeclarationStartingWithTemplate(Context, DeclEnd);
     break;
+    // HLSL Change Begin.
+  case tok::kw_export:
+    // Ignore export for now.
+    ConsumeToken();
```

Comment:
Maybe we should emit a warning that the export is ignored instead of silently consuming it?